src/CheckoutSdk.Extensions/CheckoutSdk.Extensions.csproj

```diff
@@ -1,28 +1,35 @@
 <Project Sdk="Microsoft.NET.Sdk">
-    <PropertyGroup>
-        <PackageId>CheckoutSDK.Extensions.Microsoft</PackageId>
-        <Title>Checkout.com SDK Extension</Title>
-        <Description>Checkout.com Extension for Microsoft Configuration and Dependency Injection</Description>
-        <AssemblyTitle>CheckoutSDK.Extensions</AssemblyTitle>
-        <NeutralLanguage>en-GB</NeutralLanguage>
-        <Authors>Checkout.com</Authors>
-        <Copyright>Checkout.com</Copyright>
-        <PackageProjectUrl>https://github.com/checkout/checkout-sdk-net</PackageProjectUrl>
-        <License>https://github.com/checkout/checkout-sdk-net/raw/master/LICENSE.md</License>
-        <Icon>https://raw.githubusercontent.com/checkout/checkout-sdk-net/master/assets/checkout-icon.png</Icon>
-        <PackageTags>Checkout.com;payments;gateway;sdk</PackageTags>
-        <RepositoryType>git</RepositoryType>
-        <RepositoryUrl>git://github.com/checkout/checkout-sdk-net</RepositoryUrl>
-        <TargetFrameworks>net5.0;netstandard2.0;netcoreapp3.1</TargetFrameworks>
-        <RootNamespace>CheckoutSDK.Extensions.Configuration</RootNamespace>
-    </PropertyGroup>
-    <ItemGroup>
-      <ProjectReference Include="..\CheckoutSdk\CheckoutSdk.csproj" />
-    </ItemGroup>
-    <ItemGroup>
-      <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="3.1.22" />
-      <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="3.1.22" />
-      <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="3.1.22" />
-    </ItemGroup>
+  <PropertyGroup>
+    <PackageId>CheckoutSDK.Extensions.Microsoft</PackageId>
+    <Title>Checkout.com SDK Extension</Title>
+    <Description>Checkout.com Extension for Microsoft Configuration and Dependency Injection</Description>
+    <AssemblyTitle>CheckoutSDK.Extensions</AssemblyTitle>
+    <NeutralLanguage>en-GB</NeutralLanguage>
+    <Authors>Checkout.com</Authors>
+    <Copyright>Checkout.com</Copyright>
+    <PackageProjectUrl>https://github.com/checkout/checkout-sdk-net</PackageProjectUrl>
+    <License>https://github.com/checkout/checkout-sdk-net/raw/master/LICENSE.md</License>
+    <Icon>https://raw.githubusercontent.com/checkout/checkout-sdk-net/master/assets/checkout-icon.png</Icon>
+    <PackageTags>Checkout.com;payments;gateway;sdk</PackageTags>
+    <RepositoryType>git</RepositoryType>
+    <RepositoryUrl>git://github.com/checkout/checkout-sdk-net</RepositoryUrl>
+    <TargetFrameworks>net6.0;net5.0;netstandard2.0</TargetFrameworks>
```

Comment:
Please add `6.0.x` to the files described below so that .NET 6 is installed and available in these workflows.
* .github/workflows/build-master.yml
* .github/workflows/build-pull-request.yml
* .github/workflows/build-release.yml

Reply:
Hi Raul, that has been done. I have squashed the commits into one and the PR should now merge and build properly with the current CI/CD pipeline. Thanks again for your time and when I can get the integration tests to run I will see if I can find a decent solution to the polymorphic serialization in STJ

O

Reply:
Np, thanks!